source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -79,10 +79,13 @@
                             window.location = urlGenerator.action(calificationPeriod, "Evaluations");
                         }
                     }
-                },
-                error: function () {
-                    alert('Fallo interno. Por favor recargue la página.');
                 }
+            })
+            .fail(function () {
+                alert('Fallo interno. Por favor recargue la página.');
```

Comment:
Probably it will not fail (what is bad) because there is a redirect and the final result is a HTML. In other calls it fails because we expect an JSON and de-serialization fails.

I was asking because before this release it was an active bug, and I am not sure if it has been totally resolved.